README-PLUGINS.md

```diff
@@ -0,0 +1,92 @@
+# nbgitpuller - downloader plugin documentation
+
+nbgitpuller uses [pluggy](https://pluggy.readthedocs.io/en/stable/) as a framework
+to load any installed nbgitpuller-downloader plugins. There are three downloader plugins
+available right now:
+- [nbgitpuller-downloader-googledrive](https://github.com/jupyterhub/nbgitpuller-downloader-googledrive)
+- [nbgitpuller-downloader-dropbox](https://github.com/jupyterhub/nbgitpuller-downloader-dropbox)
+- [nbgitpuller-downloader-generic-web](https://github.com/jupyterhub/nbgitpuller-downloader-generic-web)
+
+
+There are several pieces to be aware of for the plugin to work correctly:
+1. The setup.cfg(or setup.py) file must have the entry_points definition.
+For example:  
+> [options.entry_points]  
+nbgitpuller =  
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;dropbox=nbgitpuller_downloader_dropbox.dropbox_downloader
```

Comment:
I suggest the use of a code block, with `toml` formatted language, and indentation for being part of the numbered list, and I assume that `&nbsp;` entries are a mistake.

````suggestion

   ```toml
   [options.entry_points]  
   nbgitpuller = dropbox=nbgitpuller_downloader_dropbox.dropbox_downloader
   ```
````